src/overview/overview.tsx

```diff
@@ -1,15 +1,8 @@
 import React from 'react'
 
-interface OverviewProps {
-  readonly overview: {
-    readonly companyName: string
-    readonly symbol: string
-    readonly website: string
-    readonly description: string
-  }
-}
+import { OverviewContainerStateProps } from './overviewContainer'
 
-const Overview: React.SFC<OverviewProps> = ({ overview }) =>
+const Overview: React.SFC<OverviewContainerStateProps> = ({ overview }) =>
   overview === null ? null : (
```

Comment:
I don't think overview would be null now, since the initial state is 
`{
  companyName: '',
  symbol: '',
  website: '',
  description: ''
}`
this is the same  in keystat, maybe we can put a loading component or something else later on